gbpservice/neutron/tests/unit/nfp/configurator/api/v1/controllers/test_controller.py

```diff
@@ -0,0 +1,315 @@
+import mock
+import unittest
+import subprocess
+
+from pecan import rest
+from pecan import set_config
+from pecan.testing import load_test_app
+
+mock_request = mock.Mock()
+
+from gbpservice.neutron.nfp.configurator.api.v1.controllers import controller
+
+
+class ControllerTestCase(unittest.TestCase, rest.RestController):
```

Comment:
The class and file names should have configurator name part of it. Otherwise, it looks more generic. In future, if we add more controllers, it will be difficult to retain this code as is.